flowmachine/flowmachine/core/query.py

```diff
@@ -676,19 +643,27 @@ def _db_store_cache_metadata(self):
                     "SELECT * FROM cache.cached WHERE query_id='{}'".format(self.md5)
                 )
             )
+
             with con.begin():
+                cache_record_insert = """
+                INSERT INTO cache.cached 
+                (query_id, version, query, created, access_count, last_accessed, compute_time, 
```

Comment:
I was going to say that you never compute it again, but that's not necessarily true.. I think we can leave that for a followup tho, as it would require some rethinking of how queries are identified as cached and what the cache.cached table does.